diff --git a/pack.js b/pack.js
index e690cb08e667ceeea60d155c7d99aa373fcf467b..15205018c4bc0925286b039ae15cbe9dc9c95cd3 100644
--- a/pack.js
+++ b/pack.js
@@ -51,6 +51,9 @@ const writePackageJsonContent = (0, Function_1.pipe)(TE.do, TE.bind("content", (
     carry("peerDependencies", content, packageJson);
     carry("gitHead", content, packageJson);
     carry("bin", content, packageJson);
+    packageJson["tsplus"] = {
+        typeDir: "./"
+    }
     const exports = {};
     const mainExports = {};
     if (fs.existsSync(`./build/mjs/index.mjs`)) {
