diff --git a/pack.js b/pack.js
index 988c1552544f433ba5ce6f8622f50ff820508b15..1b1119b8b9d48d80468a052586deaa7d99b45a07 100644
--- a/pack.js
+++ b/pack.js
@@ -47,6 +47,7 @@ const writePackageJsonContent = (0, Function_1.pipe)(TE.do, TE.bind("content", (
     carry("peerDependencies", content, packageJson);
     carry("gitHead", content, packageJson);
     carry("bin", content, packageJson);
+    carry("tsplus", content, packageJson);
     const exports = {};
     const mainExports = {};
     if (fs.existsSync(`./build/mjs/index.mjs`)) {
